README.md

```diff
@@ -50,66 +50,76 @@ The project is structured around multiple components:
 
 # Getting started
 
-## Requirements
+## Platform Requirements
 
-- `python3`
-- `python3-venv`
+- The setup requires a Gen-6 or newer Intel CPU (for Intel PT) and sufficient
+  RAM to run several VMs at once.
+- A modifed Linux host kernel is required for VM-based snapshot fuzzing with
+  Intel PT coverage feedback. This setup does not run inside a VM or container!
 
-~~~
+## Local Installation
+
+The userspace installation and fuzzing workflow has been tested for recent
+Ubuntu (>=20.04) and Debian (>=bullseye). The base installation is captured
+as an Ansible workflow which you can bootstrap using Python:
+
+~~~sh
 sudo apt-get install python3 python3-venv
+make deploy
 ~~~
 
-## Setup
+Ansible setup will ask for your root password.
+If you are using a _passwordless sudo_ setup, just skip this by pressing enter.
 
-kAFL's deployment offers the possibility of local or remote installation.
+## Remote Installation
 
-In both cases, you will find in the installation directory:
-- `.env` file: useful environment variables for your scripts
-- `.venv` Python virtual environment: where kAFL fuzzer is installed
+kAFL's deployment offers the possibility of remote installation using Ansible.
+Update the `deploy/inventory` file according to the [Ansible inventory
+guide](https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html)
+and make sure to **remove** the first line:
 
+> localhost ansible_connection=local
 
-### Local
 
-- installation directory: `<repo_root>/kafl`
+Deployment will install kAFL to `$HOME/kafl` of the target machines:
 
-Run the depployment with:
-~~~
+~~~sh
 make deploy
 ~~~
 
-You will be prompted for your root password.
-If you are using a _passwordless sudo_ setup, just skip this by pressing enter.
+Note:
+- If your nodes require a proxy setup, update the `group_vars/all.yml`.
+- Remote deployment assumes password-less ssh and sudo.
```

Comment:
The remote deployment relies on Ansible's default options, so it won't ask for an SSH password (but this can be done with `--ask-pass`) and it doesn't require a sudo installation, this is managed by the [become](https://docs.ansible.com/ansible/latest/user_guide/become.html) directives in the playbook.

It's misleading to indicate that.

Reply:
Ok, I was not sure and assume we're not really testing it with remote password-sudo. Can remove or point to the BECOME article.